docs/release-notes.md

```diff
@@ -1,9 +1,50 @@
 # Release notes
 
-## Trino Gateway 10 (24 July 2024)
+## Trino Gateway 11 (5 Sep 2024)
+
+[JAR file gateway-ha-11-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/11/gateway-ha-11-jar-with-dependencies.jar),
+Docker container `trinodb/trino-gateway:11`
+
+* [:warning: Breaking change:](#breaking) Require Java 22 for build and runtime.
+  ([#441](https://github.com/trinodb/trino-gateway/pull/441))
+* Add support for using an external service with a REST API to process requests
+  and implement any custom routing rules.
+  ([#423](https://github.com/trinodb/trino-gateway/pull/423))
+* Add support to optionally deactivate adding `X-Forwarded-*` HTTP headers to
+  each request with the `addXForwardedHeaders: false` configuration in 
+  `routing`.
+  ([#417](https://github.com/trinodb/trino-gateway/pull/417))
+* Add OpenMetrics endpoint to
+  [enable monitoring](operation.md#monitoring) with Prometheus and compatible
+  systems.
+  ([#429](https://github.com/trinodb/trino-gateway/pull/429))
+* Add support to [deactivate hostname verification for the certificate of the
+  Trino clusters](security.md#cert-trino).
+  ([#436](https://github.com/trinodb/trino-gateway/pull/436))
+* Add option to use additional, custom `/v1/statement`-like paths as
+  Trino client REST API endpoints on the Trino clusters behind Trino Gateway.
+  ([#326](https://github.com/trinodb/trino-gateway/pull/326))
+* Allow using Spanner as backend database.
```

Comment:
Do we somehow need to mention this in the docs .. I kinda think not .. its really very much an edge case and not something we really support. Thoughts @willmostly @oneonestar @Chaho12

Reply:
I'd prefer not mention it. We didn't even test it on Spanner.

Reply:
Never used Spanner and never tested it so.. i think it is difficult to say that we allow using Spanner.
Better safe to say nothing

Reply:
Okay .. i will add a sentence at the end that points to the milestone for all merged PRs.